tapdb/sqlc/migrations/000031_asset_transfer_label.up.sql

```diff
@@ -0,0 +1,2 @@
+-- Add a column `label` to table `asset_transfer`.
+ALTER TABLE asset_transfers ADD COLUMN label VARCHAR DEFAULT '';
```

Comment:
Should this be `DEFAULT NULL` instead? Since `sqlStr()` will turn an empty string into a `NULL` value.